oak_functions/examples/weather_lookup/module/src/tests.rs

```diff
@@ -158,3 +162,55 @@ fn test_location_from_slice() {
     assert_eq!(bytes, value.to_bytes(),);
     assert_eq!(value, Location::from_bytes(&bytes));
 }
+
+// This benchmark test takes a very long time, so is not run during tests.
+// To actually run the bench, use `cargo bench --features run-internal-bench`.
+#[cfg_attr(feature = "run-internal-bench", bench)]
+#[cfg_attr(not(feature = "run-internal-bench"), allow(dead_code))]
+fn bench_wasm_handler(bencher: &mut Bencher) {
+    let mut manifest_path = std::env::current_dir().unwrap();
+    manifest_path.push("Cargo.toml");
+    let wasm_module_bytes =
+        test_utils::compile_rust_wasm(manifest_path.to_str().expect("Invalid target dir"))
+            .expect("Couldn't read Wasm module");
+    let mut rng = rand::thread_rng();
+    let buf = generate_and_serialize_sparse_weather_entries(&mut rng, 10_000).unwrap();
+    let entries = parse_lookup_entries(buf).unwrap();
+    let lookup_data = Arc::new(LookupData::for_test(entries));
+
+    let summary = bencher.bench(|bencher| {
+        let logger = Logger::for_test();
+        let wasm_handler = WasmHandler::create(
+            &wasm_module_bytes,
+            lookup_data.clone(),
+            Arc::new(None),
+            logger,
+            None,
+        )
+        .expect("Couldn't create the server");
+        let rt = tokio::runtime::Runtime::new().unwrap();
+        bencher.iter(|| {
+            let request = Request {
+                body: br#"{"lat":52.1,"lon":-0.1}"#.to_vec(),
+            };
+            let resp = rt
+                .block_on(wasm_handler.clone().handle_invoke(request))
+                .unwrap();
+            assert_eq!(resp.status, StatusCode::Success as i32);
+        });
+    });
+
+    if let Some(summary) = summary {
+        println!("Summary statistics: {:?}", summary);
```

Comment:
Why is it doing `println!` and not `info!` ?

Reply:
I don't think there is a logger installed during tests

Reply:
This was just part of debugging to see min and max values. Now removed.